ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -2329,252 +2330,139 @@ MiUseLargeDriverPage(IN ULONG NumberOfPtes,
     return FALSE;
 }
 
-ULONG
+VOID
 NTAPI
-MiComputeDriverProtection(IN BOOLEAN SessionSpace,
-                          IN ULONG SectionProtection)
+MiSetSystemCodeProtection(
```

Comment:
Yes, we can not, but that would mean, that we are either stuck with formatting that doesn't conform to our coding style forever, as well as the use of pseudo-annotations, or we reformat entire files at once, which would mean somebody would suddenly be blamed for large parts of the file, even though he has not contributed to it in functional way. And since I rewrote the entire function, I see no point in just keeping the non-conforming style of the function header alone.